launcher/modplatform/ModIndex.cpp

```diff
@@ -70,11 +70,18 @@ auto ProviderCapabilities::hash(ResourceProvider p, QIODevice* device, QString t
     }
 
     QCryptographicHash hash(algo);
-    if(!hash.addData(device))
+    if (!hash.addData(device))
         qCritical() << "Failed to read JAR to create hash!";
 
     Q_ASSERT(hash.result().length() == hash.hashLength(algo));
     return { hash.result().toHex() };
 }
 
+QString getMetaURL(ResourceProvider provider, QString slug)
+{
+    return ((provider == ModPlatform::ResourceProvider::FLAME) ? "https://www.curseforge.com/minecraft/mc-mods/"
+                                                               : "https://modrinth.com/mod/") +
+           slug.remove(".pw.toml");
+}
```

Comment:
nope, I created this function because I did not have a metadata object: when exporting I do not create the Metadata but have access to the provider and slug.
The slug for some reason when comes from files is having that suffix so I just need to remove it in order to obtain the slug used by CurseForge. Do you want to check id it's a suffix with `endsWith` and remove it only then?

Reply:
Oh, you are passing in the slug.
I expected it to not have the suffix, i guess it's just a misleading field name